spec/dor/indexing_service_spec.rb

```diff
@@ -80,23 +80,22 @@
 
     it 'should reindex the pids and not commit by default' do
       pids = [1..10].map(&:to_s)
-      pids.each { |pid| expect(Dor::IndexingService).to receive(:reindex_pid).with(pid, nil, false) }
+      pids.each { |pid| expect(Dor::IndexingService).to receive(:reindex_pid).with(pid, raise_errors: false) }
       expect(@mock_solr_conn).to_not receive(:commit)
       Dor::IndexingService.reindex_pid_list pids
     end
 
     it 'should reindex the pids and commit if should_commit is true' do
       pids = [1..10].map(&:to_s)
-      pids.each { |pid| expect(Dor::IndexingService).to receive(:reindex_pid).with(pid, nil, false) }
+      pids.each { |pid| expect(Dor::IndexingService).to receive(:reindex_pid).with(pid, raise_errors: false) }
       expect(ActiveFedora.solr).to receive(:conn).and_return(@mock_solr_conn)
       expect(@mock_solr_conn).to receive(:commit)
       Dor::IndexingService.reindex_pid_list pids, true
     end
 
     it 'should proceed despite individual indexing failures' do
-      pids = [1..10].map(&:to_s)
-      expect(Dor::IndexingService).to receive(:reindex_pid).with(pids[0], nil, false)
-      pids[1..-1].each { |pid| expect(Dor::IndexingService).to receive(:reindex_pid).with(pid, nil, false) }
+      pids = (1..10).map(&:to_s)
+      pids.each { |pid| expect(Dor::IndexingService).to receive(:reindex_pid).with(pid, raise_errors: false) }
```

Comment:
i am... not quite sure what i was thinking when i wrote it the way it was originally.  👍 on this simplification.